examples/meta/generator/targets/cpp.json

```diff
@@ -119,7 +119,7 @@
             "get_option": "$object->get<std::string>($arguments)",
             "get_string": "$object->get<std::string>($arguments)"
         },
-        "StaticCall": "C$typeName::$method($arguments)",
+        "StaticCall": "$typeName::$method($arguments)",
```

Comment:
C prefix will be dropped anyways, and there is are no static calls now except for ShogunEnv, so I saw no harm

Reply:
I think This should rather be done when the prefix is dropped, not yet

Reply:
sure, but CShogunEnv is ugly though :"D